app/subjects/comment-form.cjsx

```diff
@@ -58,11 +58,7 @@ module.exports = React.createClass
     if @state.subjectDefaultBoard
       <QuickSubjectCommentForm {...@props} subject={@props.subject} user={@props.user} />
     else
-      <p>
-        There is no default board for subject comments setup yet, Please{' '}
-        <button className="link-style" onClick={=> @setState(tab: 1)}>start a new discussion</button>{' '}
-        or {@linkToClassifier('return to classifying')}
-      </p>
+      @notSetup() # The project has not initialized default boards. Therefore render the notSetup message.
```

Comment:
can you give this function a more descriptive name instead of leaving a comment to explain what it does

Reply:
Also where is the function notSetup actually defined? I don't see it in this PR.

Reply:
It's an existing method on the component: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/subjects/comment-form.cjsx#L54-L55

Functions can be components, in recent versions of React, so this method could perhaps be extracted out to a small helper component, which is basically a function that renders a paragraph of text..